src/test/scala/org/openhorizon/exchangeapi/route/organization/TestPostOrgChangesRoute.scala

```diff
@@ -324,10 +324,10 @@ class TestPostOrgChangesRoute extends AnyFunSuite with BeforeAndAfterAll with Be
   // Resource Change that is dynamically needed, specific to the test case.
   //only does one at a time for ease of deleting when finished
   def fixtureResourceChange(testCode: ResourceChangeRow => Any, testData: ResourceChangeRow): Any = {
-    try{
+    //try{
```

Comment:
This introduces a moderate bug. In the case of a exception thrown in the call stack the test library will not have a method of failing the testcase. The `Try` block provides that function. Do to the nature of Change records we intentionally do not clean them up in the fixture (where the warning comes from), but clean them up in aggregate in the `afterAll()`.

Reply:
Fixed.